### PR TITLE
Add external script run from sceewlog during EEW

### PR DESCRIPTION
**Pull request by FMassin**
* Add external script parameter and implementation in sceewlog
* Add external script example `sceewlog2file`
* Update docs and changelog
 
# Test for the module
The external script setup in `sceewlog.cfg:EEW.script` is executed with argument `test=1` on startup:
```bash
date && sceewlog --debug -H host.docker.internal:18180 -u test -d mysql://sysop
:sys0pm4rn@host.docker.internal:3335/seiscomp3
Mon Jul 31 18:08:08 UTC 2023
sceewlog2file test=1 > /tmp/sceewlog-script.log 2> /tmp/sceewlog-script.err &
18:08:09 [debug] Adding plugin path: .
18:08:09 [debug] Adding plugin path: /root/.seiscomp/plugins
18:08:09 [debug] Adding plugin path: /opt/seiscomp/lib/plugins
18:08:09 [debug] Adding plugin path: /opt/seiscomp/lib
18:08:09 [debug] Adding plugin path: /opt/seiscomp/share/plugins
18:08:09 [debug] Adding plugin path: /opt/seiscomp/share/plugins/sceewlog
18:08:09 [debug] Trying to open plugin at /opt/seiscomp/share/plugins/dbmysql.so
18:08:09 [info] Plugin dbmysql registered
18:08:09 [info] 
Plugins:
--------
 [1]
  description: MySQL database driver
       author: GFZ Potsdam <seiscomp-devel@gfz-potsdam.de>
      version: 0.9.2
          API: 15.1.0

18:08:09 [info] Connect to messaging
18:08:09 [debug] Trying to connect to test@host.docker.internal:18180 with primary group = 
18:08:09 [debug] Attempt to connect to host.docker.internal:18180/production
18:08:10 [debug] Registered clientname: test
18:08:10 [info] Outgoing messages are encoded to match schema version 0.12
18:08:10 [info] Want to flush 0 backlog messages
18:08:10 [debug] Subscribed to group EVENT
18:08:10 [debug] Subscribed to group LOCATION
```

# Test the external script
The provided external script `sceewlog2file` can be fully tested as follows:
```bash
> sceewlog2file magID=mag/42 type=Meew author=author42 magnitude=4.2 lat=10 lon=-88 depth=4.2 nstor
g=4 nstmag=2 ts=2023-01-01T04:02:42.4200 tsobject=2023-01-01T04:02:42.4200 diff=4.2 ot=2023-01-01T04:02:42.4200 eew=1
Successfuly read the lalo2textbnaFile /opt/seiscomp/share/sceewlog/EEW_reference.bna
lat: 10.0 and lon: -88.0 are within polygon: Frente a Costa de Nicaragua
2023-01-01 04:02:42 > 2022-12-31 22:02:42-06:00
Succesfully wrote to file: /tmp/.tweet.json
```

### CODE REVIEW

File: apps/eew/sceewlog/sceewlog.py [hunk 1]
```diff
@@ -227,6 +229,23 @@ def initConfiguration(self):
...
+        except Exception as e:
+            seiscomp.logging.warning('Not possible to read EEW.script. Setting this to None')
+            self.eewScript = None
+
+        if self.eewScript is not None :
+            logprefix = '/tmp/sceewlog-script.'
+            print("%s test=1 > %slog 2> %serr &" % ( self.eewScript, 
+                                                           logprefix,
+                                                           logprefix ))
+            os.system( "%s test=1 > %slog 2> %serr &" % ( self.eewScript,
```
Comment (billyburgoa) [inline apps/eew/sceewlog/sceewlog.py:245]: Fred, to avoid blocking the main thread, please, consider using subprocess instead of os.system

Comment (FMassin) [reply apps/eew/sceewlog/sceewlog.py:245]: Of course it is not blocked at all ! There is a `&` !!! It releases instantly!

Comment (FMassin) [reply apps/eew/sceewlog/sceewlog.py:245]: Hey, I have added my tests in the PR description ...

Comment (billyburgoa) [reply apps/eew/sceewlog/sceewlog.py:245]: > Of course it is not blocked at all ! There is a `&` !!! It releases instantly! Try it! And I don't want to add dependencies to the main sceewlog.

👍Okay, I didn't check that detail

**Pull request merged**